src/specs/api.yaml

```diff
@@ -185,6 +185,24 @@ paths:
         in: path
         required: true
         description: ID of the experiment to find.
+    post:
```

Comment:
This is missing the responses in case of a 403 or a 401.

Reply:
I added the 401 only, because the 403 isn't a possibility in the case of a POST (we are creating a new resource, not trying to access an existing one)